mm2src/docker_tests.rs

```diff
@@ -913,6 +932,15 @@ mod docker_tests {
         log!([block_on(enable_native(&mm_bob, "MYCOIN1", vec![]))]);
         log!([block_on(enable_native(&mm_alice, "MYCOIN", vec![]))]);
         log!([block_on(enable_native(&mm_alice, "MYCOIN1", vec![]))]);
+        log!("Get MYCOIN/MYCOIN1 orderbook on Alice side to trigger subscription");
```

Comment:
As far as I understand, `setprice` RPC request triggers subscription too. Why this is required here?

Reply:
You're right, worth to note that `setprice` is not called on Alice side, it is `buy` which also triggers the subscription to the corresponding topic. Will remove this part.

Reply:
Tried to remove it and got failing test due to following:
1. At the moment of `setprice` call Alice node is not yet subscribed to topic so the new order is not received immediately by her.
2. Buy call does not trigger orderbook request (and won't trigger).
3. `MakerOrderUpdated` message is simply ignored now if order does not exist on node. It is done intentionally to let the orders state sync later on `PubkeyKeepAlive` received.

I will leave a TODO here, this orderbook request can be removed when your part is finished.

Reply:
Ok thank you :)